### PR TITLE
Add sentry to form

### PR DESCRIPTION
**Pull request by adamclark-dev**
Add sentry tracking to form

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -12,6 +13,7 @@ function App() {
         <Route exact path="/" component={Home} />
         <Route path="/question/:question_number" render={props => <Question {...props} />} />
         <Route path="/outcome/:outcome_number" render={props => <OutcomeMessage {...props} />} />
+        <Raven dsn="https://174a79db7cf54ad2a03bafd2a075af8b@sentry.io/482571" />
```
Comment (pvhee) [inline src/App.jsx]: doesn't this need to be injected via the pipeline? or will all apps (local,  staging etc) track this in the same way?

**Comment by AndyEPhipps** [comment]
@adamclark-dev : is this just catching errors? Do I need to do anything else to integrate this?

File: src/App.jsx (hunk 1 above)
Comment (adamclark-dev) [reply src/App.jsx]: All of the stages share the same dsn, it then automagically detects environment.

**Comment by adamclark-dev** [comment]
@AndyEPhipps no need to do any integration other than this.

File: src/App.jsx (hunk 1 above)
Comment (AndyEPhipps) [reply src/App.jsx]: ✨ Automagically ✨

**Pull request merged**